### PR TITLE
LPS-135729 Adjust tabletoolbar's panel styles

### PR DESCRIPTION
**Pull request by diegonvs**
This PR depends on https://github.com/brianchandotcom/liferay-portal/pull/104894

It fixes the truncated text in table toolbar panel.

### Before:

Screen Shot 2021-07-30 at 19 20 16

### After:

Screen Shot 2021-07-30 at 18 50 01

Screen Shot 2021-07-30 at 19 00 23

Screen Shot 2021-07-30 at 19 00 30

### AlloyEditor's style:

Screen Shot 2021-07-30 at 19 13 55

### There is one issue with this PR:

Screen Shot 2021-07-30 at 20 07 32

~~Due to a fixed value of `width`, there is an empty space in the panel depending how many items we have.~~

~~I tried to use `min-height` as alloy-editor does but We still setting a defined height and We might still have some empty spaces there.~~

This issue is also reproduced in combopanel usages, like the format combo:

Screen Shot 2021-07-30 at 20 12 13

### CODE REVIEW

**Comment by diegonvs** [comment]
I fixed by creating another class instead of using `*combopanel` and setting `min-height` and `min-width` values.  :)

Screen Shot 2021-08-02 at 15 04 54

**Comment by diegonvs** [comment]
Fixed for tabletoolbar panels but "This issue is also reproduced in combopanel usages, like the format combo" still being reproduced. Should I apply this change to `cke_combopanel` too?

File: modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/tabletoolbar/plugin.js [hunk 1]
```diff
@@ -954,7 +954,9 @@
 
 				let panelBlock;
 
-				panel.className = `${panel.className ?? ''} cke_combopanel`;
+				panel.className = panel.className
+					? `${panel.className} lfr-cke_table_panel`
+					: 'lfr-cke_table_panel';
```
Comment (markocikos) [inline modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/tabletoolbar/plugin.js:959]: It looks like `panel.className` is always undefined at this point, with class inserting logic done later. So, we can simplify to:
```js
panel.className = 'lfr-table-panel'
```

I renamed the class there for consistency:
- `cke` prefix is used by ckeditor. Our classes should not contain it.
- our classes should have only dashes (`-`)

**Review by markocikos — CHANGES_REQUESTED** [review]
@diegonvs  See comment inline. Also, please rebase before pushing; Github thinks there is a conflict, although there really isn't one.

**Comment by markocikos** [comment]
> Fixed for tabletoolbar panels but "This issue is also reproduced in combopanel usages, like the format combo" still being reproduced. Should I apply this change to `cke_combopanel` too?

Do we want to modify these styles for all panels? If so, you should add a liferay-specific class to `editable`, and modify styles on something like`.lfr-editable .cke-panel`.

File: modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/tabletoolbar/plugin.js (hunk 1 above)
Comment (diegonvs) [reply modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/tabletoolbar/plugin.js:959]: > It looks like panel.className is always undefined at this point, with class inserting logic done later. So, we can simplify to:

I was thinking if someone could place a className in panel definition but it's a premature optimization. Removing!

> cke prefix is used by ckeditor. Our classes should not contain it.
> our classes should have only dashes (-)

Oh, I didn't know it. Changing! :D

**Comment by diegonvs** [comment]
> Do we want to modify these styles for all panels? If so, you should add a liferay-specific class to editable, and modify styles on something like.lfr-editable .cke-panel.

cke_combopanel is just [being used in richcombo usages](https://github.com/ckeditor/ckeditor4/blob/62e3ae4c0dd1ddb37882ca0cfcf8078a23b986b1/plugins/richcombo/plugin.js#L93). My idea is to fix this empty space bug in the combopanels like the format one. However, it will just being reproduced in table headers richcombo, where there are just few options.

See table headers richcombo:
Screen Shot 2021-08-03 at 14 30 24

Another rich combo but containing more options:
Screen Shot 2021-08-03 at 14 28 17

**Comment by diegonvs** [comment]
ci:test:sf

**Comment by diegonvs** [comment]
ci:test:relevant

**Comment by diegonvs** [comment]
```
   [exec] > Task :dxp:apps:commerce-salesforce-connector:commerce-salesforce-connector:compileJava FAILED
     [exec] 
     [exec] > Task :dxp:apps:document-library-opener:document-library-opener-onedrive-web:compileJava
   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/commands/20210803113441814
     [exec] 
     [exec] > Task :apps:headless:headless-discovery:headless-discovery-web:packageRunBuild
     
```

unrelated, re-running

**Comment by diegonvs** [comment]
ci:test:relevant

**Review by markocikos — APPROVED** [review]
Looks good, but there is a conflict. Please fix and forward after.

**Comment by diegonvs** [comment]
Rebased, force pushed and then forwarding!

**Comment by diegonvs** [comment]
ci:forward

**Comment by diegonvs** [comment]
ci:forward

**Comment by diegonvs** [comment]
ci:forward

**Comment by markocikos** [comment]
Failure is unrelated. The same is in https://github.com/liferay-frontend/liferay-portal/pull/1342 and https://github.com/liferay-frontend/liferay-portal/pull/1344

**Comment by diegonvs** [comment]
ci:test:forward

**Comment by diegonvs** [comment]
ci:forward

**Comment by diegonvs** [comment]
Screen Shot 2021-08-11 at 14 58 35

6b1rsath69751

**Comment by diegonvs** [comment]
Closing in favor of https://github.com/brianchandotcom/liferay-portal/pull/105433

**Pull request closed without merging**